cli/src/commands/backout.rs

```diff
@@ -78,17 +86,20 @@ pub(crate) fn cmd_backout(
         )
     };
     let mut new_base_tree = merge_commit_trees(tx.repo(), &parents)?;
+
     for commit_to_back_out in to_back_out {
-        let commit_to_back_out_subject = commit_to_back_out
-            .description()
-            .lines()
-            .next()
-            .unwrap_or_default();
-        let new_commit_description = format!(
-            "Back out \"{}\"\n\nThis backs out commit {}.\n",
-            commit_to_back_out_subject,
-            &commit_to_back_out.id().hex()
-        );
+        let new_commit_description = {
+            let template = tx.parse_commit_template(ui, &template_text)?;
```

Comment:
Since `commit_to_backout` exists in the base repo, we don't have to re-parse template with the current mut repo. Perhaps, we can build `Vec` of descriptions to get around borrowing issue of `WorkspaceCommandHelper`.

Reply:
Updated, PTAL.